app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/screen/TeleconsultRecordUiRenderer.kt

```diff
@@ -11,6 +12,15 @@ class TeleconsultRecordUiRenderer(
     ui.setTeleconsultationType(model.teleconsultationType)
     ui.setPatientTookMedicines(model.patientTookMedicines)
     ui.setPatientConsented(model.patientConsented)
+    renderDoneProgress(model.teleconsultRecordCreateState)
+  }
+
+  private fun renderDoneProgress(teleconsultRecordCreateState: TeleconsultRecordCreateState?) {
+    if (teleconsultRecordCreateState == CREATING_TELECONSULT_RECORD) {
```

Comment:
Can we extract this logic to a computed property on the model, say `isCreatingTeleconsultRecord`?